src/components/molecules/Preview/Preview.tsx

```diff
@@ -0,0 +1,30 @@
+import styles from './Preview.module.scss';
+
+import { Table } from 'nhsuk-react-components';
+import concatClassNames from '@/utils/concatClassNames';
+import { PreviewProps } from './Preview.types';
+
+export function Preview({ preview }: PreviewProps): JSX.Element {
+  return (
+    <Table
+      className={concatClassNames(styles.preview, 'nhsuk-u-margin-bottom-4')}
```

Comment:
TODO: not a fan of using the nhsuk-u-margin-bottom-4 on the component. Should pass it down.